Wishlist/wishlist.html

```diff
@@ -0,0 +1,137 @@
+                <div class="icon-items"><a href="../index.html">Home</a></div>
+                <div class="icon-items"><a href="./wishlist.html"><i class="far fa-heart"></i></a></div>
+                <div class="icon-items"><a href="../Cart/cart.html"><i class="fas fa-shopping-cart"></i></a></div>
+            </div>
+        </div>
+        <div class="wishlist-items">
+            <div class="merch-content">
+                <div class="content-head"><h3>Wishlist <i class="far fa-heart"></i></h3></div>
+                <div class="merch-cards">
+                    <div class="card-item horizontal">
+                        <div class="card-info">
+                            <div class="card-image">
+                                <img src="../Assets/Tommy-Hilfiger.jpg" alt="image">
+                            </div>
+                            <div class="card-infoText">
+                                <div class="card-author">
+                                    <h1>Keyboard</h1>   
+                                    <p>Powerful & Mechanical</p>
+                                </div>
+                                <div class="card-content">
+                                   <del>100$</del> <span>69$</span> 
+                                </div>
+                            </div>
+                        </div>
+                        <div class="card-buttons">
+                            <span>Move to Bag</span>
+                        </div>
+                        <div class="cross-btn"><i class="far fa-times"></i></div>
+                    </div>
+                    <div class="card-item horizontal">
+                        <div class="cross-btn"><i class="far fa-times"></i></div>
+                        <div class="card-info">
+                            <div class="card-image">
+                                <img src="../Assets/Tommy-Hilfiger.jpg" alt="image">
+                            </div>
+                            <div class="card-infoText">
+                                <div class="card-author">
+                                    <h1>Keyboard</h1>   
+                                    <p>Powerful & Mechanical</p>
+                                </div>
+                                <div class="card-content">
+                                   <del>100$</del> <span>69$</span> 
+                                </div>
+                            </div>
+                        </div>
+                        <div class="card-buttons">
+                            <span>Move to Bag</span>
+                        </div>
+                    </div>
+                    <div class="card-item horizontal">
+                        <div class="cross-btn"><i class="far fa-times"></i></div>
+                        <div class="card-info">
+                            <div class="card-image">
+                                <img src="../Assets/Tommy-Hilfiger.jpg" alt="image">
```

Comment:
In image tag, you can use --> loading="lazy" <--- . It will improve the app performance